hack/quickstart-container.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+set -e
+
+CINCINNATI_REPO="${CINCINNATI_REPO:?Please set CINCINNATI_REPO}"
+CINCINNATI_REGISTRY="${CINCINNATI_REGISTRY:?Please set CINCINNATI_REGISTRY}"
+
+CINCINNATI_IMG="${CINCINNATI_IMG:-cincinnati-img}"
+CINCINNATI_IMG_TAG="${CINCINNATI_IMG_TAG:-quickstart}"
+CINCINNATI_IMG_TAGGED="${CINCINNATI_IMG}:${CINCINNATI_IMG_TAG}"
+CINCINNATI_GB="cincinnati-graph-builder"
+CINCINNATI_GB_PORT="${CINCINNATI_GB_PORT:-8080}"
+CINCINNATI_GB_CREDENTIALS="${CINCINNATI_GB_CREDENTIALS:-}"
+CINCINNATI_PE="cincinnati-pe"
+CINCINNATI_PE_PORT="${CINCINNATI_PE_PORT:-8081}"
+CINCINNATI_LABEL="cincinnati-$RANDOM"
+
+CURL_OUT="$(mktemp)"
+
+# Build a container image with both components
+if ! 2>&1 >/dev/null docker image inspect "${CINCINNATI_IMG_TAGGED}" || \
+    test "${FORCE_REBUILD}"; then
+    IMAGE="${CINCINNATI_IMG}" IMAGE_TAG="${CINCINNATI_IMG_TAG}" dist/build_deploy.sh
+fi
+
+function cleanup() {
+    set +e
+    containers=$(docker ps --quiet --filter "label=${CINCINNATI_LABEL}")
+    if test "${containers}"; then
+        docker stop ${containers} 2>&1 >/dev/null
+    fi
+    rm -f "${CURL_OUT}"
+    pkill "docker logs -f ${CINCINNATI_GB}"
+}
+trap cleanup EXIT
+
+# Run the graph builder
+echo Spawning graph-builder container...
+docker run -d --rm \
+  --label="${CINCINNATI_LABEL}" \
+  --name "${CINCINNATI_GB}" \
+  --env "RUST_LOG=actix_web=error,dkregistry=trace" \
```

Comment:
trace seems aggressive.

Reply:
This is meant for demo purposes as it shows nicely what requests are being made. I'm fine to change that to whatever you suggest

Reply:
I lowered it to error too